walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,39 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
same with this method, consider making it static if you don't need state in this class. By state I mean, any member variables or any data that you need to hold